VideoWeb/VideoWeb/ClientApp/src/app/on-the-day/quick-links/quick-links.component.ts

```diff
@@ -83,7 +96,14 @@ export class QuickLinksComponent implements OnInit, OnDestroy {
         let errorsFound = false;
 
         if (this.quickLinkNameFormControl.invalid) {
-            this.error.nameError = true;
+            if (this.quickLinkNameFormControl.errors['emptyOrWhitespaceError']) {
+                this.error.notEmptyOrWhitespaceError = true;
+                this.error.nameError = true;
```

Comment:
can we move this line above line 99, then we wont have to repeat it in the next if block nested here

Reply:
thats done now